Lib/shutil.py

```diff
@@ -541,13 +541,17 @@ def _rmtree_unsafe(path, onerror):
     try:
         with os.scandir(path) as scandir_it:
             entries = list(scandir_it)
+    except FileNotFoundError:
+        entries = []
```

Comment:
Why not just return?

Reply:
I made the requested change.  Please review again.